js/base/script/script.js

```diff
@@ -0,0 +1,52 @@
+//min
+
+const MY_NAME = 'Oleh';
+const TEACHER_NAME = 'Ros';
+let studentName = 'Petro';
+const BRO_NAME = 'Andriy';
+
+
+/*
+const 4MY_NAME = 'Helo';
+const TEACHER NAME = 'Sor';
+let @studentName = 'Ortep';
+const --BRO-NAME = 'Yirdna';
+*/
+
+let camelCase;
+let snake_case;
+
+
+
+// normal
+
+// 1
+let yourName = prompt('Enter Your Name!');
+alert(`Hello, ${yourName}`);
+
+// 2
+let birthYear = prompt ('Enter year of your birth');
+const yourAge = 2022 - birthYear;
+alert(`You are ${yourAge} or ${yourAge - 1} years old.`);
+
+// 3
+let sqrSide = prompt("Enter a number and i'll multiply it by 4!");
```

Comment:
Well done. Also, you can use a little tricky thing - `+prompt('...')`. It will try to convert a value from what the user enters to a Number. I will show it later